docs/tutorials/upgrade-extension.md

```diff
@@ -28,42 +28,40 @@ kind: ClusterExtension
 metadata:
   name: argocd
 spec:
+  namespace: argocd
+  serviceAccount:
+    name: argocd-installer
   source:
     sourceType: Catalog
     catalog:
       packageName: argocd-operator
       version: 0.5.0
-  install:
-    namespace: argocd
-    serviceAccount:
-      name: argocd-installer
 ```
 
 * Update the version field in the ClusterExtension resource:
 
     ``` terminal
-      kubectl apply -f - <<EOF
-      apiVersion: olm.operatorframework.io/v1alpha1
-      kind: ClusterExtension
-        metadata:
-          name: argocd
-        spec:
-          source:
-            sourceType: Catalog
-            catalog:
-              packageName: argocd-operator
-              version: 0.6.0 # Update to version 0.6.0
-          install:
-            namespace: argocd
-            serviceAccount:
-              name: argocd-installer
-      EOF
+    kubectl apply -f - <<EOF
+    apiVersion: olm.operatorframework.io/v1alpha1
+    kind: ClusterExtension
+    metadata:
+      name: argocd
+    spec:
+      namespace: argocd
+      serviceAccount:
+        name: argocd-installer
+      source:
+        sourceType: Catalog
+        catalog:
+          packageName: argocd-operator
+          version: 0.6.0 # Update to version 0.6.0
+    EOF
```

Comment:
The indention is wrong that if we cut and past will not work

```
$ kubectl apply -f - <<EOF
>     apiVersion: olm.operatorframework.io/v1alpha1
>     kind: ClusterExtension
>     metadata:
>       name: argocd
>     spec:
>       namespace: argocd
>       serviceAccount:
>         name: argocd-installer
>       source:
>         sourceType: Catalog
>         catalog:
>           packageName: argocd-operator
>           version: 0.6.0 # Update to version 0.6.0
>     EOF
> 

```

Then, also we have a problem with the content/specs

```
$ kubectl apply -f - <<EOF
> apiVersion: olm.operatorframework.io/v1alpha1
> kind: ClusterExtension
> metadata:
>   name: argocd
> spec:
>   namespace: argocd
>   serviceAccount:
>     name: argocd-installer
>   source:
>     sourceType: Catalog
>     catalog:
>       packageName: argocd-operator
>       version: 0.6.0 # Update to version 0.6.0
> EOF
Error from server (BadRequest): error when creating "STDIN": ClusterExtension in version "v1alpha1" cannot be handled as a ClusterExtension: strict decoding error: unknown field "spec.namespace", unknown field "spec.serviceAccount"
```

Reply:
Copy and paste worked for me, applying this block of yaml also worked for me without issue.

Reply:
I got the latest changes from main and worked 
False alarm sorry.